pkg/danmep/danmep.go

```diff
@@ -139,8 +141,16 @@ func SetDanmEpSysctls(ep danmtypes.DanmEp) error {
     podNs.Close()
     origNs.Set()
   }()
-  // set sysctls for IPv6 (since IPv6 is enabled on all interfaces, let's disable on those where it is not needed)
-  if ep.Spec.Iface.AddressIPv6 == "" {
+  // set sysctls on those interfaces where IPv6 is needed
+  if ep.Spec.Iface.AddressIPv6 != "" {
```

Comment:
this looks like now a lot of specific code, which is not easiyl extendable

how about a little rework like this:
- we maintain the following data structure: a function reference returning a bool, and a list of stuff to set
- the sysctl setting code iterates through the sysctl records, call the referenced function, if that return true we blindly set what is in the list